### PR TITLE
feat(reachableservices): add docs about auto reachable services

### PR DESCRIPTION
**Pull request by slonka**
closes https://github.com/kumahq/kuma-website/issues/1503

rendered: https://deploy-preview-1508--kuma.netlify.app/docs/dev/production/upgrades-tuning/fine-tuning/#config-trimming-by-using-meshtrafficpermission 

Add docs for auto reachable services

A general question to the reviewers - what do you think should be here and which details could be left in MADR only?

Did you sign your commit? [Instructions](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md#sign-your-commits) yes

Have you read [Contributing guidelines](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md)? yes

### CODE REVIEW

File: app/_src/production/upgrades-tuning/fine-tuning.md [hunk 1]
```diff
@@ -16,6 +16,109 @@ The result is that:
...
+{% warning %}
+1. This feature only works with [MeshTrafficPermission](/docs/{{ page.version }}/policies/meshtrafficpermission),
+   if you're using [TrafficPermission](/docs/{{ page.version }}/policies/traffic-permissions) you need to migrate to MeshTrafficPermission,
+   otherwise enabling this feature could stop all traffic flow.
+2. Due to [a bug](https://github.com/kumahq/kuma/issues/6589) [ExternalServices](/docs/{{ page.version }}/policies/external-services) won't work without Traffic Permissions without [Zone Egress](/docs/{{ page.version }}/production/cp-deployment/zoneegress), if you're using External Services you need to keep associated TrafficPermissions.
+   {% endwarning %}
+
+Starting with release 2.5 the problem stated in [reachable services](#reachable-services) section
+can be also mitigated by defining [MeshTrafficPermissions](/docs/{{ page.version }}/policies/meshtrafficpermission) in combination with:
+- `KUMA_EXPERIMENTAL_AUTO_REACHABLE_SERVICES=true` environment variable set on **global CP** (for helm use `--set {{site.set_flag_values_prefix}}controlPlane.envVars.KUMA_EXPERIMENTAL_AUTO_REACHABLE_SERVICES=true`) and
```
Comment (lobkovilya) [inline app/_src/production/upgrades-tuning/fine-tuning.md:31]: why on global? should be on a zone, no?

Comment (slonka) [reply app/_src/production/upgrades-tuning/fine-tuning.md:31]: you're right 🤦

Comment (slonka) [inline app/_src/production/upgrades-tuning/fine-tuning.md:31]:
```suggestion
- `KUMA_EXPERIMENTAL_AUTO_REACHABLE_SERVICES=true` environment variable set on **zone CP** (for helm use `--set {{site.set_flag_values_prefix}}controlPlane.envVars.KUMA_EXPERIMENTAL_AUTO_REACHABLE_SERVICES=true`) and
```

File: app/_src/production/upgrades-tuning/fine-tuning.md [hunk 2]
```diff
@@ -16,6 +16,109 @@ The result is that:
...
+You can combine `autoReachableServices` with [reachable services](#reachable-services), but **reachable services** will take precedence.
+{% endtip %}
+
+Sections below highlight the most important aspects of this feature, if you want to dig deeper please take a look at the [MADR](https://github.com/kumahq/kuma/blob/master/docs/madr/decisions/031-automatic-rechable-services.md#automatic-reachable-services).
+
+### Supported targetRef kinds
+
+The following kinds affect the graph generation and performance:
+- all levels of `MeshService`
+- [top](/docs/{{ page.version }}/policies/targetref/#target-resources) level `MeshSubset` and `MeshServiceSubset` with `k8s.kuma.io/namespace`, `k8s.kuma.io/service`, `k8s.kuma.io/port` labels
```
[the remark this answers is not in the document]
Comment (slonka) [reply app/_src/production/upgrades-tuning/fine-tuning.md:71]:
```suggestion
- [top](/docs/{{ page.version }}/policies/targetref/#target-resources) level `MeshSubset` and `MeshServiceSubset` with `k8s.kuma.io/namespace`, `k8s.kuma.io/service`, `k8s.kuma.io/port` tags
```

File: app/_src/production/upgrades-tuning/fine-tuning.md [hunk 3]
```diff
@@ -16,6 +16,109 @@ The result is that:
...
+- [from](/docs/{{ page.version }}/policies/targetref/#target-resources) level `MeshSubset` and `MeshServiceSubset` with all labels
+
+If you define a MeshTrafficPermission with other kind, like this one:
+
+```yaml
```
[the remark this answers is not in the document]
Comment (slonka) [reply app/_src/production/upgrades-tuning/fine-tuning.md:76]:
```suggestion
```policy_yaml
```

**Review by lobkovilya — APPROVED** [review]

**Pull request merged**